pom.xml

```diff
@@ -67,45 +67,57 @@
+        <com.googlecode.json-simple.version>1.1.1</com.googlecode.json-simple.version>
         <com.jayway.jsonpath.version>2.4.0</com.jayway.jsonpath.version>
         <com.mesosphere.marathon-client.version>0.6.2</com.mesosphere.marathon-client.version>
         <com.rabbitmq.version>5.8.0</com.rabbitmq.version>
+        <com.schibsted.spt.data.jslt>0.1.9</com.schibsted.spt.data.jslt><!--When updating the version. Change here too - https://github.com/JobService/job-service/blob/develop/docs/pages/en-us/Job-Types.md-->
         <com.squareup.okhttp.version>2.7.5</com.squareup.okhttp.version>
         <com.squareup.retrofit.version>1.9.0</com.squareup.retrofit.version>
+        <com.sun.jersey.version>1.19</com.sun.jersey.version>
         <commons-codec.version>1.14</commons-codec.version>
         <commons-io.version>2.6</commons-io.version>
         <hamcrest-all.version>1.3</hamcrest-all.version>
-        <!--Hibernate version below is updated,but the latest version is throwing exception-->
-        <!--<hibernate-validator.version>6.1.2.Final</hibernate-validator.version>-->
-        <hibernate-validator.version>5.4.3.Final</hibernate-validator.version>
+        <hibernate-validator.version>5.4.3.Final</hibernate-validator.version><!-- Latest version 6.1.2.Final is throwing exception-->
         <io.swagger.swagger-annotations.version>1.6.0</io.swagger.swagger-annotations.version>
         <jakarta.validation-api.version>2.0.2</jakarta.validation-api.version>
+        <jakarta.ws.rs-api.version>2.1.6</jakarta.ws.rs-api.version>
+        <jakarta.xml.bind-api.version>2.3.3</jakarta.xml.bind-api.version>
         <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
         <javax.mail.version>1.5.0-b01</javax.mail.version>
         <javax.validation.validation-api.version>2.0.1.Final</javax.validation.validation-api.version>
         <javax.ws.rs-api.version>2.1.1</javax.ws.rs-api.version>
+        <javax.ws.rs.jsr311-api.version>1.1.1</javax.ws.rs.jsr311-api.version>
+        <javax.xml.bind.jaxb-api.version>2.3.1</javax.xml.bind.jaxb-api.version>
         <jboss-logging.version>3.4.1.Final</jboss-logging.version>
         <joda-time.version>2.10.5</joda-time.version>
         <junit.jupiter.version>5.6.0</junit.jupiter.version>
-        <mockito.version>3.3.3</mockito.version>
+        <mockito.version>1.10.19</mockito.version> <!--Error with Latest Version 3.3.3-->
         <org.apache.httpcomponents.version>4.4.13</org.apache.httpcomponents.version>
+        <org.codehaus.jettison.version>1.4.0</org.codehaus.jettison.version>
+        <org.glassfish.javax.el.version>3.0.1-b11</org.glassfish.javax.el.version>
         <org.json.version>20190722</org.json.version>
         <org.projectlombok.version>1.18.12</org.projectlombok.version>
         <org.slf4j.version>1.7.30</org.slf4j.version>
+        <org.webjars.npm.swagger-ui-dist.version>3.25.0</org.webjars.npm.swagger-ui-dist.version>
```

Comment:
This one seems strange to me since I thought we stopped using it when we moved to https://github.com/MicroFocus/swagger-ui.

Reply:
@rahul-kulkarni73 Where is this still being used?